ProjectManagmentSystem/src/Modules/DashboardModule/components/Dashboard/Dashboard.tsx

```diff
@@ -4,15 +4,35 @@ import axios from "axios";
 import { AuthContext } from "../../../Context/AuthContext";
 import { Chart as ChartJS, ArcElement, Tooltip, Legend } from "chart.js";
 
-import {  Pie } from 'react-chartjs-2';
+import { Pie } from "react-chartjs-2";
 ChartJS.register(ArcElement, Tooltip, Legend);
 
+interface TaskData {
+  toDo: number;
+  inProgress: number;
+  done: number;
+}
+
+interface ActiveUserData {
+  activatedEmployeeCount: number;
+  deactivatedEmployeeCount: number;
+}
+
 export default function Dashboard() {
   const [usersList, setUsersList] = useState([]);
   const [projectsList, setProjectsList] = useState([]);
   const [tasksList, setTasksList] = useState([]);
   const { requestHeaders, baseUrl }: any = useContext(AuthContext);
-  const [taskData, setTaskData] = useState([]);
+  const [taskData, setTaskData] = useState<TaskData>({
+    toDo: 0,
+    inProgress: 0,
+    done: 0,
+  });
+
+  const [isAciveUser, setActiveUser] = useState<ActiveUserData>({
```

Comment:
I don't like this name as it indicates that the stored value is boolean. Something like `employeeStatusCounts` would be better.

Reply:
Done